### ISSUE TITLE
Buttons do not support aria-expanded

### ISSUE THREAD

**Issue by steverep** [material-components/material-web#3245]
**Describe the bug**
The APIs for `mwc-button` and `mwc-icon-button` have no way to set the `aria-expanded` attribute on the `<button>` element.  This is needed, for example, when the button is used to open a popup menu.  It should be set to true or false based on user input, and omitted when null or the empty string.

**To Reproduce**
n/a

**Expected behavior**
Button components should support all potential ARIA attributes as defined in the W3C spec and transfer those attributes and states to the internal `<button>` element.

**Screenshots**
n/a

**Browser Version (please complete the following information):**
n/a

**Additional context**
[ARIA 1.2 Authoring Practice for menu buttons](https://www.w3.org/TR/wai-aria-practices/#menubutton)
